src/components/movie_subinfo.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import noImg from '../images/image.png';
```

Comment:
You should be able to update this line to: 
import noImg from 'images/image.png';